### ISSUE TITLE
[Win10] Can't access the server from internet (option enabled)

### ISSUE THREAD

**Issue by Gionames** [kzahel/web-server-chrome#91]
Hello, in the router dashboard I forwarded TCP port 8887 to the PC IP where the server is running. I enabled the internet access but the remote PC (where I keyed in the server's DDNS_name:8887 and then the public_ip:8887) can't access the web server. Do I have to modify other settings? In Windows Firewall maybe?

Thanks in advance for your help,

**Comment by kzahel** [comment]
Try turning off windows firewall. Also you could make sure it's working on the local network at least, by trying to access the local network URL from another computer.

**Comment by Gionames** [comment]
Hi kzahel,
I tried on loopback and it works (127.0.0.1:8887)
Turning off Firewall does nothing
And the router firewall is turned off as well.
Any idea?

**Comment by kzahel** [comment]
Yes loopback will work.
But what about trying from another computer or phone on the local network, and accessing on the local network IP. can you at least get that working?

**Comment by Gionames** [comment]
Just tried but it won't work. On a related note, under Web Server URL(s), only the loopback shows up

**Comment by Gionames** [comment]
After "powercycling" the"Web server: started" button, I managed to have LAN IP show up and actually now the server works on the LAN. But still nothing from the internet side

**Comment by Gionames** [comment]
Ok nailed it. After the stopping and restarting the server, I stopped Firewall again and voila the server worked on the public IP (although the download speed is only 25mbps, while theoretically it should be around 100mbps). Now what is the traffic that I should allow through Windows Firewall?

**Comment by kzahel** [comment]
Very interesting. Try enabling ipv6 in the options and see if you get
another URL after restarting the server

**Comment by kzahel** [comment]
Oh I just saw you got it working. Nevermind. Probably the speed is slower
than it should be because the app is not optimised for fast reading from
disk yet

**Comment by Gionames** [comment]
All of the IP adressess of the PC shows up now, included eg. those of the VMware Workstation NICs.
 But as I said, at the moment the access from internet requires Windows Firewall to be shut off. Any idea how to fix this?

**Comment by kzahel** [comment]
I believe chrome apps on windows used to open firewall ports, but they don't anymore.
I'm not sure how to open only a specific port on windows, but you could try googling it.

**Comment by Gionames** [comment]
What do you think is the process or exe file that the 8887 port needs to be
forwarded to? Chrome.exe?

Il 20 Feb 2017 21:45, "Kyle Graehl" <notifications@github.com> ha scritto:

**Comment by kzahel** [comment]
Yes that sounds like it would work. Or maybe Chrome helper, if there's an exe like that.

**Comment by calvin-smith** [comment]
Leaving a post for others that run into this issue later on and also find this thread.

I had a Windows Firewall rule set up for ports, but it was not working.  What did the trick for me was adding chrome.exe to the "allowed apps" list.
